src/core/server/models/story/index.ts

```diff
@@ -30,6 +33,8 @@ import {
 export * from "./counts";
 export * from "./helpers";
 
+const ACTIVE_STORIES_LIMIT = 25;
+
```

Comment:
This maximum is already reflected via the provided constraint on the GraphQL schema.